ddtrace/internal/runtime/runtime_metrics.py

```diff
@@ -79,8 +98,14 @@ def __init__(self, interval=_get_interval_or_default(), tracer=None, dogstatsd_u
             self.dogstatsd_url or ddtrace.internal.agent.get_stats_url()
         )
         self.tracer: ddtrace.trace.Tracer = tracer or ddtrace.tracer
-        self._runtime_metrics: RuntimeMetrics = RuntimeMetrics()
-        self._platform_tags: List[str] = self._format_tags(PlatformTags())
+        if "runtime_metrics" in ddtrace.config._experimental_features_enabled:
```

Comment:
For setting this experimental behaviors, I was thinking of two potential ways that differ from what you have here:
1. Align on the same flag as Node.js: In [code](https://github.com/DataDog/dd-trace-js/pull/816) it's `experimental.runtimeId` though I can't tell if there's an environment variable flag
2. Use the newly proposed `DD_TRACE_EXPERIMENTAL_ENABLED` flag (as you're already doing) but specifically look for the config `DD_RUNTIME_METRICS_ENABLED` in the value

Reply:
Should we use `DD_TRACE_EXPERIMENTAL_ENABLED` for both enabling gauge metrics and runtime-id. I would like to avoid introducing two experimental feature flags.

Reply:
Since Node.js does have a separate toggle for the runtime-id, I think it would make sense to have two separate environment variables:
- `DD_TRACE_EXPERIMENTAL_ENABLED` enables gauge metrics
- An additional experimental runtime-id flag

Then if we determine the downside is small from enabling the runtime-id "by default", we can lump that in with `DD_TRACE_EXPERIMENTAL_ENABLED`. I understand the hesitation to have multiple configuration keys, but we were already going to have the one `DD_TRACE_EXPERIMENTAL_ENABLED` flag so I think it would be okay to have one more for the runtime-id

Reply:
Cool. I'll do that